apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -1315,113 +1311,25 @@ func (m *ModelManagerAPI) ModifyModelAccess(ctx context.Context, args params.Mod
 			result.Results[i].Error = apiservererrors.ServerError(errors.Annotate(err, "could not modify model access"))
 			continue
 		}
+		err = m.accessService.UpdatePermission(ctx, access.UpdatePermissionArgs{
```

Comment:
Excellent, I think this will fix the problems with my PR for getting the information about users with access to the model.